skyportal/models/obj.py

```diff
@@ -205,6 +205,7 @@ class Obj(Base, conesearch_alchemy.Point):
     offset = sa.Column(
         sa.Float, default=0.0, doc="Offset from nearest static object [arcsec]."
     )
+    t0 = sa.Column(sa.Float, nullable=True, index=True, doc="Time of first detection.")
```

Comment:
I think the description should be slightly different, as it really is just a reference time for this object, which could be a first detection or a GRB time or something else.